desk/app/groups.hoon

```diff
@@ -1496,8 +1459,37 @@
     ::
         %del
       =.  cabals.group  (~(del by cabals.group) sect)
-      =.  cor  (verify-group-cabals [flag ~ group] cor)
-      go-core
+      =/  old-sect=(set sect:g)  (sy sect ~)
+      =.  go-core
+        ::  remove from members as needed
+        ::
+        %-  go-fleet-del-sects
+        :_  old-sect
+        %+  roll
+          ~(tap by fleet.group)
+        |=  [[s=ship =vessel:fleet:g] ships=(set ship)]
+        ?.  (~(has in sects.vessel) sect)  ships
+        (~(put in ships) s)
```

Comment:
If we're already iterating over the entire fleet here, do we not just want to do the `vessel(sects (~(dif in sects.vessel) sects))` from `+go-fleet-del-sects` in-line here? ie:

```suggestion
        %-  ~(run by fleet.group)
        |=  [s=ship =vessel:fleet:g]
        vessel(sects (~(dif in sects.vessel) sects))
```